### ISSUE TITLE
Game crashes when talking with voice_loopback enabled

### ISSUE THREAD

**Issue by HunterCZ122** [LAGonauta/metahook#3]
] al_version
Meta Audio 9a51dda3e2a3cdb8d97fbc53e40b0f8e93c9eca9
 OpenAL Device: Speaker (Sound Blaster EVO Zx)
 OpenAL Version: 1.1

] version
Protocol version 48
Exe version 1.1.2.2 (valve)
Exe build: 16:10:54 Aug 10 2018 (7960)

Launch parameters: metahook.exe -steam -game valve -nofbo +al_doppler 0.3

How to reproduce:
1) Create listen server
2) Type voice_loopback 1 to console
3) Press voice button and start talking
boom.. game crashes or freezes.

**Comment by LAGonauta** [comment]
Yeah, voice support still needs work. I tested it here and I could hear me for some seconds before it crashed. I will take a look.
Does it only crash with voice_loopback, or also from people online?

**Comment by LAGonauta** [comment]
@HunterCZ122 
Can you test with the newest release?

**Comment by HunterCZ122** [comment]
@LAGonauta
Sorry for my inactivity (busy in school).
For some reason I have another problem right now. Metahook seems to not working for me. I launched metahook.exe with your parameters and it just doesn't work, I can't execute any commands related to metahook or spot any sound difference. Older metahook versions are also not working for me.
Maybe these problems are only on my end, so I will try my best to fix it.

**Comment by lespaul64** [comment]
@LAGonauta @hzqst This issue coming back in build 9920 of CS1.6 which current steam official.
For "steam_legacy" branch all is okay.
